src/sentry/integrations/github/client.py

```diff
@@ -164,8 +164,16 @@ def authorize_request(self, prepared_request: PreparedRequest) -> PreparedReques
 
         return prepared_request
 
+    def is_error_fatal(self, error: Exception) -> bool:
+        if error.response.text:
+            for msg in ["suspended"]:
```

Comment:
Do we expect to add more to this list? I think for now you can just combine the two lines so it's just `if "suspended" in error.response.text` and if we need to add more later it can be rewritten.